.github/workflows/format.yaml

```diff
@@ -62,6 +62,13 @@ jobs:
       - name: show diff
         run: git add .; git diff --exit-code HEAD
 
+      - name: setup terraform
```

Comment:
There shouldn't be any more terraform in this repo, so this should be unnecessary.

Reply:
Ah, right.  This was hanging out for a bit, and I was just trying to finish up stuff.  Will remove the terraform part